src/imgui/Shortcuts.cc

```diff
@@ -0,0 +1,60 @@
+#include "Shortcuts.hh"
+#include "imgui_internal.h"
+
+#include "SettingsConfig.hh"
+
+#include <array>
+
+namespace openmsx {
+
+static constexpr auto defaultShortcuts = std::array{
+	Shortcuts::Data{ShortcutIndex::GOTO_MEMORY_ADDRESS, ImGuiMod_Ctrl | ImGuiKey_G, ShortcutType::LOCAL},
+	Shortcuts::Data{ShortcutIndex::GOTO_DISASM_ADDRESS, ImGuiMod_None, ShortcutType::LOCAL},
+};
+
+Shortcuts::Shortcuts(SettingsConfig& config_)
+	: config(config_)
```

Comment:
The only use of the `config` member variable seems to be in this constructor. So there's no need to actually store it as a member.

And maybe it's clearer to not even pass `SettingsConfig` as a parameter. Instead maybe ImGuiManager could setup the connection between `Shortcuts` and `SettingsConfig`?  I'm not sure yet.